blazor/rich-text-editor/webassembly-performance.md

```diff
@@ -0,0 +1,132 @@
+
+You can prevent this re-rendering of the RichTextEditor component by calling the **PreventRender** method.
+
+In the following example:
+
+* **ValueChange** event is bound with a callback method, when the editor content gets changed on every time the event is completed the **StateHasChanged** will be invoked for the parent component.
+
+```cshtml
+@using Syncfusion.Blazor.RichTextEditor
+
+<SfRichTextEditor @ref="rteObj">
+    <RichTextEditorToolbarSettings Items="@Tools" Type="ToolbarType.Expand" />
+    <RichTextEditorEvents ValueChange="OnValueChange"/>
+</SfRichTextEditor>
+
+@code {
+    SfRichTextEditor rteObj
+    public void OnValueChange(Syncfusion.Blazor.RichTextEditor.ChangeEventArgs args) {
+        rteObj.PreventRender();
+    };
+    private List<ToolbarItemModel> Tools = new List<ToolbarItemModel>() 
+    { 
+        new ToolbarItemModel() { Command = ToolbarCommand.Bold }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Italic }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Underline }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.StrikeThrough }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.FontColor }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.BackgroundColor }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Formats }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Alignments }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.OrderedList }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.UnorderedList }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Outdent }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Indent }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator },
+        new ToolbarItemModel() { Command = ToolbarCommand.CreateLink }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Image }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.CreateTable }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Separator }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Undo }, 
+        new ToolbarItemModel() { Command = ToolbarCommand.Redo } 
+    }; 
+}
+```
+
+N> * **PreventRender** method internally overrides the **ShouldRender** method of the RichTextEditor to prevent rendering.
+<br/> * It is recommended to use **PreventRender** method for user interactive events such as OnCellClick, OnEventClick etc. for better performance.
+<br/> * For events without any argument such as **DataBound**, you can use **PreventRender** method of the RichTextEditor to disable rendering.
```

Comment:
<br/> * For events without any argument such as **DataBound**, you can use the **PreventRender** method of the RichTextEditor to disable rendering.